apps/web/src/SettingsRoutes.tsx

```diff
@@ -32,7 +33,7 @@ export const useSettingsRoutes = () => {
         <Route path={ROUTES.ORGANIZATION} element={<ApiKeysCard />} />
         <Route path={ROUTES.TEAM_SETTINGS} element={<MembersInvitePageNew />} />
         <Route path={ROUTES.BILLING} element={<BillingRoutes />} />
-        <Route path={ROUTES.WEBHOOK} element={<EmailSettings />} />
+        <Route path={ROUTES.WEBHOOK} element={<WebhookPage />} />
```

Comment:
It is using `/:env` in the URL to differentiate between envs.
But this param is not used in the hook to fetch the relevant details.
Also, currently it allows to visit any random URL param for example: `http://localhost:4200/settings/webhook/hello-joel` still opens the webhook page

Reply:
There is no way to access the webhook page for the `PROD` env

Reply:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Great catch -- thank you!

I updated it with a hook to force redirects (and applied it to the API Keys as well)‏